src/MCPClient/lib/clientScripts/extract_zipped_transfer.py

```diff
@@ -87,20 +87,9 @@ def call(jobs):
                 isBag = args.bag
 
                 basename = os.path.basename(target)
-                basename = basename[: basename.rfind(".")]
-
                 destinationDirectory = os.path.join(processingDirectory, basename)
 
-                # trim off '.tar' if present (os.path.basename doesn't deal well with '.tar.gz')
-                try:
-                    tar_extension_position = destinationDirectory.rindex(".tar")
-                    destinationDirectory = destinationDirectory[:tar_extension_position]
-                except ValueError:
-                    pass
-
-                zipLocation = os.path.join(
-                    processingDirectory, os.path.basename(target)
-                )
+                destinationDirectory = get_extract_dir_name(destinationDirectory)
```

Comment:
Looking good, we just need to make sure `get_extract_dir_name(..)` is imported at the top of the file:

* `from fileOperations import get_extract_dir_name`